src/notification/index.js

```diff
@@ -77,6 +78,10 @@ export const getAccountFromNotificationData = (
       realm_uri,
       parsed_url: realmUrl,
       match_count: urlMatches.length,
+      known_urls: knownUrls,
+      unique_identities_count: urlMatches
+        .map(matchIndex => `${identities[matchIndex].realm.origin} ${identities[matchIndex].email}`)
+        .filter((value, index, self) => self.indexOf(value) === index).length,
```

Comment:
This line is clever! But also quadratic. In this context the list is unlikely to be long enough that that matters; but nevertheless writing something linear-time, like `new Set(…).size`, would make me feel more at ease.